sympy/matrices/matrices.py

```diff
@@ -3549,6 +3549,28 @@ def _diagonalize_clear_subproducts(self):
         del self._eigenvects
 
     def jordan_cell(self, eigenval, n):
+        """
+        Generate Jordan Blocks if eigenvalue and dimension is provided.
+
+        https://en.wikipedia.org/wiki/Jordan_matrix
+
+        Examples
+        ========
+        >>> from sympy.matrices import jordan_cell
```

Comment:
Ok, looks like this got me a bit confused. Is ```jordan_cell``` in ```sympy.matrices.matrices.py``` some kind of helper method then? They are roughly similar but the one in ```matrices.py``` actually imports ```MutableMatrix``` as well.